dali/kernels/signal/decibel/decibel_calculator.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_SIGNAL_DECIBEL_DECIBEL_CALCULATOR_H_
+#define DALI_KERNELS_SIGNAL_DECIBEL_DECIBEL_CALCULATOR_H_
+
+#include <cmath>
+
+namespace dali {
+namespace kernels {
+namespace signal {
+
+template <typename T>
+struct DecibelCalculator {
+ public:
+  explicit DecibelCalculator(T mul = 10.0, T s_ref = 1.0, T min_ratio = 1e-8)
+      : mul_log2_(mul / std::log2(10.0))
```

Comment:
```suggestion
      : mul_log2_(mul * std::log10(2.0))
```
...and we could have this as a constant - it's a pity it's not defined in math.h

Reply:
done